src/migrations/201602231400_initial.js

```diff
@@ -22,17 +22,10 @@ function createFulfillmentsTable (knex) {
   })
 }
 
-function createEntryGroupsTable (knex) {
-  return knex.schema.createTableIfNotExists('entry_groups', table => {
-    table.increments()
-    table.timestamp('created_at').defaultTo(knex.fn.now())
-  })
-}
-
 function createEntriesTable (knex) {
   return knex.schema.createTableIfNotExists('entries', table => {
     table.increments()
-    table.integer('entry_group')
+    table.string('group', 1024)
```

Comment:
It allows you to get the balances of all accounts at any point in time and ensure that they sum to a constant value.

If they were only collected by `created_at` and not `group`, it would include extra entries, so it wouldn't sum to a constant.

If they were collected by `transfer_id` you wouldn't be able to perform the operation at any point in time, only after any transfer completes.